### PR TITLE
Fixes #1556:Fix GithubProfileScraper test

### LINKED ISSUE #1556: GithubProfileScraper test fails
- Issue type: Bug report

### Short description

Here is the log from build - 

```java
org.loklak.TestRunner > org.loklak.api.search.GithubProfileScraperTest.githubProfileScraperOrgTest FAILED
    org.junit.ComparisonFailure: expected:<http[s]://fossasia.org> but was:<http[]://fossasia.org>
        at org.junit.Assert.assertEquals(Assert.java:115)
        at org.junit.Assert.assertEquals(Assert.java:144)
        at org.loklak.api.search.GithubProfileScraperTest.githubProfileScraperOrgTest(GithubProfileScraperTest.java:53)
```

### Environment

 - Operating system: Ubuntu 16.04LTS
 - Software version: 8d521a8461ef65a4b4ab5c8b90d5f5b43d0fbf23
 - Software source: Github repository

### Steps to reproduce

1. Get even with `loklak:development`.
2. `./gradlew build -i`
3. Look at the logs

### Expected behaviour
The test should pass.

### Actual behaviour
Test fails.

### PR DESCRIPTION
**Pull request by S2606**
### Short description
I have:
- [X] There is a corresponding issue for this pull request.
- [X] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`
- [X] There is only strictly only one commit per issue.

### For the reviewers
I have:
- [ ] Reviewed this pull request by an authorized contributor.
- [ ] The reviewer is assigned to the pull request.

### CODE REVIEW

**Comment by S2606** [comment]
@singhpratyush @vibhcool Please review

**Comment by S2606** [comment]
@singhpratyush Done

File: test/org/loklak/api/search/GithubProfileScraperTest.java [hunk 1]
```diff
@@ -46,11 +46,11 @@ public void githubProfileScraperOrgTest() {
 
         Post fetchedProfile = githubScraper.scrapeGithub(profile, br);
 
-        assertEquals(fetchedProfile.getString("short_description"), shortDescription);
-        assertEquals(fetchedProfile.getString("user"), userName);
-        assertEquals(fetchedProfile.getString("user_id"), userId);
-        assertEquals(fetchedProfile.getString("location"), location);
-        assertEquals(fetchedProfile.getString("special_link"), specialLink);
```
Comment (vibhcool) [inline test/org/loklak/api/search/GithubProfileScraperTest.java]: see here, there are some indentation convention to leave a space after comma. Could you add the space after comma in new changes please!!

**Comment by S2606** [comment]
@vibhcool Done!

**Pull request merged**